themes/frontierline/sass/components/content-views/_content-grid.scss

```diff
@@ -0,0 +1,21 @@
+// Used for a custom static layout of content
+
+.grid-item {
+  box-sizing: border-box;
+  padding-left: 15px;
+  padding-right: 15px;
+  height: 360px;
```

Comment:
min-height is probably better. Since it isn't a robust grid, this was a workaround to make sure the boxes stay horizontally aligned.